src/charm.py

```diff
@@ -249,7 +249,7 @@ def run(self, cmd: List[str]):
         Args:
             cmd: Command to be run.
         """
-        self._container.exec(cmd)
+        self._container.exec(cmd).wait()
```

Comment:
Do we need the `.wait()`? The other PR doesn't have it.

Reply:
Yeah doesn't really work without it, The other PR exec via subprocess so we don't need it there